spec/rails_helper.rb

```diff
@@ -97,4 +97,7 @@
   config.append_after(:each) do
     DatabaseCleaner.clean
   end
+
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
```

Comment:
Already defined in `spec_helper.rb`
https://github.com/SumOfUs/Champaign/blob/development/spec/spec_helper.rb#L35-L43

Reply:
ok. I will remove that one